### PR TITLE
yoshi-java as CODEOWNERS

### PR DESCRIPTION
**Pull request by chingor13**
Leaving @pongad as codeowner on pubsub.

### CODE REVIEW

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -2,35 +2,21 @@
...
 google-cloud-firestore/src/*          @schmidt-sebastian
-google-cloud-language/src/*           @pongad
 google-cloud-logging/src/*            @saicheems @andreamlin @jabubake
-google-cloud-monitoring/src/*         @pongad
-google-cloud-pubsub/src/*             @pongad
-google-cloud-resourcemanager/src/*    @pongad
+google-cloud-pubsub/src/*             @pongad @googleapis/yoshi-java
 google-cloud-spanner/src/*            @vam-google @vkedia
 google-cloud-speech/src/*             @vam-google
 google-cloud-storage/src/*            @andreamlin @neozwu
```
Comment (garrettjonesgoogle) [inline .github/CODEOWNERS]: Remove neozwu everywhere

File: .github/CODEOWNERS [hunk 2]
```diff
@@ -2,35 +2,21 @@
...
-google-cloud-examples/src/*           @pongad
 google-cloud-firestore/src/*          @schmidt-sebastian
-google-cloud-language/src/*           @pongad
 google-cloud-logging/src/*            @saicheems @andreamlin @jabubake
-google-cloud-monitoring/src/*         @pongad
-google-cloud-pubsub/src/*             @pongad
-google-cloud-resourcemanager/src/*    @pongad
+google-cloud-pubsub/src/*             @pongad @googleapis/yoshi-java
 google-cloud-spanner/src/*            @vam-google @vkedia
 google-cloud-speech/src/*             @vam-google
```
Comment (garrettjonesgoogle) [inline .github/CODEOWNERS]: You can probably remove @vam-google everywhere - Vadym, what do you think?

File: .github/CODEOWNERS [hunk 3]
```diff
@@ -2,35 +2,21 @@
...
-google-cloud-errorreporting/src/*     @pongad
-google-cloud-examples/src/*           @pongad
 google-cloud-firestore/src/*          @schmidt-sebastian
-google-cloud-language/src/*           @pongad
 google-cloud-logging/src/*            @saicheems @andreamlin @jabubake
-google-cloud-monitoring/src/*         @pongad
-google-cloud-pubsub/src/*             @pongad
-google-cloud-resourcemanager/src/*    @pongad
+google-cloud-pubsub/src/*             @pongad @googleapis/yoshi-java
 google-cloud-spanner/src/*            @vam-google @vkedia
```
Comment (garrettjonesgoogle) [inline .github/CODEOWNERS]: remove vkedia, add @snehashah16

File: .github/CODEOWNERS [hunk 4]
```diff
@@ -2,35 +2,21 @@
...
-google-cloud-core/src/*               @pongad
 google-cloud-datastore/src/*          @neozwu @vam-google
-google-cloud-dep-verification/src/*   @pongad
-google-cloud-dlp/src/*                @neozwu
-google-cloud-dns/src/*                @pongad
-google-cloud-errorreporting/src/*     @pongad
-google-cloud-examples/src/*           @pongad
 google-cloud-firestore/src/*          @schmidt-sebastian
-google-cloud-language/src/*           @pongad
 google-cloud-logging/src/*            @saicheems @andreamlin @jabubake
```
Comment (garrettjonesgoogle) [inline .github/CODEOWNERS]: Remove saicheems and jabubake everywhere

File: .github/CODEOWNERS (hunk 3 above)
Comment (snehashah16) [reply .github/CODEOWNERS]: Lgtm

**Pull request merged**